lib/lightning_web/live/audit_live/index.ex

```diff
@@ -47,24 +47,37 @@ defmodule LightningWeb.AuditLive.Index do
   end
 
   def diff(assigns) do
-    rhs = assigns.metadata.after
+    with lhs <- assigns.metadata.before || %{},
+         rhs <- assigns.metadata.after || %{},
+         true <- lhs != %{} || rhs != %{} do
+      changes =
+        lhs
+        |> Map.keys()
+        |> Enum.concat(Map.keys(rhs))
+        |> Enum.sort()
+        |> Enum.uniq()
+        |> Enum.map(fn key ->
```

Comment:
It would make sense but not sure if that's the case, if the key for `after` only exists conditioned to an existing `before`, couldn't it just iterate the `lhs`? Would appreciate to understand this better.

Reply:
@jyeshe I wanted to try and make the display as 'failsafe' as possible, given the importance of audit events.

With that in mind, I asked myself: "If I make a mistake and I have a key in `after` that is not present in `before` (for example) what is the safest thing that the code could do?" To me the answer would be that it would be better to show this 'weird' setting rather than not showing anything at all (which is what would happen if I just iterate LHS).

By showing something, the data is still presented in the UI and so the information is still presented to the user (in case it is useful) and it s more likely that the underlying cause (key mismatch)  will be identified and corrected.

Reply:
Got it, first implementation to make sure that everything is fine 👍